Documentation/network/bgp-control-plane/bgp-control-plane-v2.rst

```diff
@@ -738,6 +738,17 @@ Similarly, ``internalTrafficPolicy`` is considered for ``ClusterIP`` advertiseme
 
 .. _bgp-override:
 
+Overlapping Advertisements
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When configuring ``CiliumBGPAdvertisement``, it is possible that two or more 
+advertisments match for a given Service.  Prior to Cilum ``1.17.TODO``, 
+overlapping matches were not expected and the last sequential match was used. 
+Today, overlapping advertisements are supported.  Overlaps are handled as 
+additive operations for attributes that can be unioned (communities). If 
+local preference was set, it remains set to the first advertisement matched.
```

Comment:
Please keep the `.. _bgp-override:` anchor right above the related subsection, do not insert your paragraph between the two.

Doc subsection looks good otherwise, thanks!

Seeing that the PR is now waiting for the next branching to happen before being merged, your `1.17.TODO` will most likely be `1.18`.